nilearn/_utils/data_gen.py

```diff
@@ -21,33 +21,84 @@
 
 
 def generate_mni_space_img(n_scans=1, res=30, random_state=0, mask_dilation=2):
-    rng = check_random_state(random_state)
+    """Generate mni space img.
+
+    Parameters
+    ----------
+    n_scans : int, optional
+        Number of scans.
+        Default=1.
+
+    res : int, optional
+        Desired resolution, in mm, of output images.
+        Default=30.
```

Comment:
Any reason for this default ? This looks really unusual.

Reply:
Indeed that is strange, maybe it's a typo and supposed to be 3

Reply:
OK. I think we should simply revert that. Sorry.